support/@trustfractal/data-host/src/popup/components/App.tsx

```diff
@@ -1,9 +1,12 @@
-// import Protocol from "@components/Protocol";
-import Protocol from './Protocol';
+import Protocol from '@components/Protocol';
+import GlobalStyle from "@styles/GlobalStyle";
 function App() {
 
   return (
+    <>
+    <GlobalStyle />
     <Protocol />
+    </>
   );
```

Comment:
Do we have a formatter configured that handles jsx? I think these are usually indented below the top-level <></>.